### PR TITLE
location fix in HMP for loocv

### PR DESCRIPTION
**Pull request by jelmerborst**
self.location was set incorrectly when provided, which introduced a problem with LOOCV where it was copied from an existing model. I do not think line 111 is used for anything else, but that should be checked.

### CODE REVIEW

File: src/hsmm_mvpy/models.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def __init__(self, data, epoch_data=None, sfreq=None, cpus=1, event_width=50, sh
         self.event_width_samples = int(np.round(self.event_width / self.steps))
         if location is None:
             self.location = int(self.event_width / self.steps//2)+1
-        else: self.location =  int(np.rint(location / self.steps))
+        else:
```
Comment (GWeindel) [inline src/hsmm_mvpy/models.py:111]: Indeed but the int(np.rint() insured that it was an int, perhaps then just add an assert?

**Review by GWeindel — APPROVED** [review]
Fine by me, we should just add the assert in case a location is specified in float

File: src/hsmm_mvpy/models.py (hunk 1 above)
Comment (jelmerborst) [reply src/hsmm_mvpy/models.py:111]: sure, or leave the `int(np.rint())` and just remove `/ self.steps` (which made the location go to 0 and it estimated overlapping events).

Do you have a preference for` int(np.rint())` or assert?

Comment (GWeindel) [reply src/hsmm_mvpy/models.py:111]: We can just keep ```int(np.rint())``` I would say

**Pull request merged**